projects/pgai/pgai/semantic_catalog/loader.py

```diff
@@ -96,15 +96,28 @@ async def load_views(con: psycopg.AsyncConnection, oids: list[int]) -> list[View
     # TODO: add support for continuous aggregates
     assert len(oids) > 0, "list of oids must not be empty"
     async with con.cursor(row_factory=dict_row) as cur:
+        await cur.execute("select 1 from pg_extension where extname = 'timescaledb'")
+        is_timescale = await cur.fetchone() is not None
```

Comment:
I debated caching this boolean value within the module as we'll probably need a similar value for `load_tables` (and others?), but ultimately decided against it for the time being given the async nature of the code, and that doing this query is largely negligible in overall runtime.

Reply:
Yeah, i think cutting corners is fine for now. We can come back later and squeeze every ounce of performance out if needed.